src/config.rs

```diff
@@ -480,44 +455,24 @@ impl fmt::Debug for SharedConfigValues {
 
 impl CliConfig {
     pub fn load_config_file_values(&mut self) {
-        let config_file_path = self.config_path.clone().or_else(get_config_file);
+        let config_file_path = match self.config_path.clone().or_else(get_config_file) {
+            Some(p) => p,
+            None => {
+                info!("No config file specified. Running with default values");
+                return;
+            }
+        };
+        info!("Loading config from {:?}", &config_file_path);
 
-        if config_file_path.is_none() {
-            info!("No config file specified. Running with default values");
-            return;
-        }
-        let unwrapped_config_file_path = config_file_path.unwrap();
-        info!("Loading config from {:?}", &unwrapped_config_file_path);
-
-        let config_file = fs::File::open(&unwrapped_config_file_path);
-        if config_file.is_err() {
-            info!(
-                "Failed to open config file at {:?}",
-                &unwrapped_config_file_path
-            );
-            return;
-        }
+        let content = match std::fs::read_to_string(config_file_path) {
+            Ok(s) => s,
+            Err(e) => {
+                info!("Failed reading config file: {}", e);
+                return;
+            }
+        };
 
-        let bufreader = std::io::BufReader::new(config_file.unwrap());
-        // serde_ini doesn't support inline comments. We treat every hashtag as a comment starter and everything that follows
-        // it as not part of the key's value.
-        // The method below will filter out any errors that occur.
-        // TODO: Is there a cleaner way to do this? One with less allocations.
-        let comment_free_lines: Vec<String> = bufreader
-            .lines()
-            .filter_map(Option::Some)
-            .map(|x| x.unwrap())
-            .map(|mut l: String| {
-                let last_index = l.rfind('#').unwrap_or_else(|| l.len());
-                l.drain(..last_index).collect()
-            })
-            // The password field takes the whole value as the password. We need to remove the space between
-            // the password and the # character.
-            .map(|l: String| l.trim().to_string())
-            .collect();
-
-        let comment_free_content = comment_free_lines.join("\n");
-        let config_content: FileConfig = serde_ini::from_str(&comment_free_content).unwrap();
+        let config_content: FileConfig = toml::from_str(&content).unwrap();
```

Comment:
I don't think it should panic here. 
Reporting the error would make much more sense

Reply:
Yeah it would be ideal for the toml errors to be propagated to the user and inform them how to fix the file.

Reply:
Feel free to request a PR on my fork

Reply:
I'd probably use ˋexceptˋ instead of unwrap and write something like "error during deserialization". The except call does propagate the error message as well do the user would have an idea what is wrong :)

Reply:
I'm happy to leave this as-is for now. Instead, it'd be good to make a proper error reporting system that can be more user-friendly than panics